src/plugins/nestedRows/data/dataManager.js

```diff
@@ -392,6 +392,28 @@ class DataManager {
     return !!(rowObj.__children && rowObj.__children.length);
   }
 
+  /**
+   * Returns `true` if the row at the provided index has a parent.
+   *
+   * @param {number} index Row index.
+   * @returns {boolean} `true` if the row at the provided index has a parent, `false` otherwise.
+   */
+  hasRowParent(index) {
```

Comment:
Maybe the `isChild` will be yet more clear? WDYT?

```suggestion
  isChild(index) {
```